specifications/Native bridge pool ERC20.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Native Bridge Pool ERC20
+author: @gusjavaz
+discussion: alicenet/issues/404
+team: Backend
+category: Development
+status: Draft
+related: alicenet/issues/423
+created: 2022-11-15
+---
+
+## Introduction
+
+#### Summary
+
+This allows ERC20 tokens to be transferred from Ethereum mainnet into Alicenet and back.
+
+#### Context
+
+Bridge Pools allow transfers of assets from one chain to another.
+Alicenet Native Bridge Pools allow transfers between Ethereum and Alicenet both ways.
+The main reasons for bridging assets into Alicenet is that transfers in Alicenet are:
+* cheaper than Ethereum
+* faster than Ethereum
+* as safe as in Ethereum
+
+#### Goals
+
+Allow for ERC20 assets to be transferred from Ethereum to Alicenet and back.
+
+#### Non Goals
+
+Do not enable transfers between Alicenet and other L2 chains
+
+#### Assumptions
+
+Smart contract functions will be called by user through Alicenet Wallet
+
+## Specification
+
+#### Overview
+
+To enable bridging for a specific ERC20 token, a particular Native Bridge Pool Implementation smart contract should be deployed for the specific ERC20 contract.
+These implementations will support multiple version and expose the following operations:
+* Initialize 
+* Deposit
+* Withdraw
+These implementations will be deployed by Bridge Pool Factory contract by selecting a specific version and will be initialized for ERC contract address.
```

Comment:
need a new line here

Reply:
Fixed and moved to https://github.com/alicenet/specs/pull/16